samples/graph-activity-feed/csharp/ActivityFeedNotification/Helpers/TeamMembersNotificationRecipient.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Newtonsoft.Json;
+
+namespace Microsoft.Graph
+{
+    //
+    // Summary:
+    //     The type TeamMembersNotificationRecipient.
+    [JsonObject(MemberSerialization = MemberSerialization.OptIn)]
+    public class TeamMembersNotificationRecipient : TeamworkNotificationRecipient
```

Comment:
I think these are not the helpers but just a class models.
We can add Models folder and put it there ?

Reply:
Fixed